resources/views/index.php

```diff
@@ -0,0 +1,4 @@
+<?php include('header.php');?>
```

Comment:
Should the header include opening the body? Just so that we don't have a <body> without a </body>?